### PR TITLE
8039261: [TEST_BUG]: There is not a minimal security level in Java Preferences and the TestApplet.html is blocked.

### PR DESCRIPTION
**Pull request by azvegint**
This fix simply removes tests mentioned in the bug with for unsupported scenarios(applets in browser).

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8039261](https://bugs.openjdk.java.net/browse/JDK-8039261): [TEST_BUG]: There is not a minimal security level in Java Preferences and the TestApplet.html is blocked.

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)
 * [Pankaj Bansal](https://openjdk.java.net/census#pbansal) (@pankaj-bansal - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3441/head:pull/3441` \
`$ git checkout pull/3441`

Update a local copy of the PR: \
`$ git checkout pull/3441` \
`$ git pull https://git.openjdk.java.net/jdk pull/3441/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3441`

View PR using the GUI difftool: \
`$ git pr show -t 3441`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3441.diff">https://git.openjdk.java.net/jdk/pull/3441.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
Please confirm that none of them are in the problem list.

**Comment by azvegint** [comment]
> Please confirm that none of them are in the problem list.

I do confirm that.

File: test/jdk/java/awt/event/KeyEvent/KeyReleasedInAppletTest/KeyReleasedInAppletTest.java [hunk 1]
```diff
@@ -1,234 +0,0 @@
-/*
```
Comment (mrserb) [inline test/jdk/java/awt/event/KeyEvent/KeyReleasedInAppletTest/KeyReleasedInAppletTest.java:1]: The "TestApplet.java" near this file is part of the test and can be removed as well.

Comment (azvegint) [reply test/jdk/java/awt/event/KeyEvent/KeyReleasedInAppletTest/KeyReleasedInAppletTest.java:1]: Fixed.

**Review by mrserb — APPROVED** [review]

**Review by pankaj-bansal — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Review by azuev-java — APPROVED** [review]

**Pull request closed without merging**